src/main/java/org/sagebionetworks/web/client/widget/entity/editor/AttachmentConfigEditor.java

```diff
@@ -46,16 +49,22 @@ public void configure(WikiPageKey wikiKey, Map<String, String> widgetDescriptor,
 		view.clear();
 		view.configure(wikiKey, dialogCallback);
 		wikiAttachments.configure(wikiKey);
+		uploadedFile = null;
+		this.fileInputWidget.configure("Browse...", new CallbackP<FileUpload>() {
+			@Override
+			public void invoke(FileUpload uploadFile) {
+				view.showUploadSuccessUI();
+				//enable the ok button
+				uploadedFile = uploadFile;
+				dialogCallback.setPrimaryEnabled(true);
+				addFileHandleId(uploadFile.getFileHandleId());
+			}			
+		});
+		//set new validation callback?
+		this.fileInputWidget.configureValidation(new TableFileValidator(), null);
```

Comment:
Why a TableFileValidator for a wiki attachment?  It should allow any file to be attached...